cloud-gcp/src/main/java/com/sequenceiq/cloudbreak/cloud/gcp/GcpPlatformParameters.java

```diff
@@ -95,6 +95,11 @@ public String resourceDefinition(String resource) {
         return cloudbreakResourceReaderService.resourceDefinition("gcp", resource);
     }
 
+    @Override
+    public String resourceDefinitionInSubDir(String subDir, String resource) {
+        return cloudbreakResourceReaderService.resourceDefinitionInSubDir(subDir, "gcp", resource);
```

Comment:
in azure this is FileReaderUtils.readFileFromClasspathQuietly("definitions" + subDir + "/azure-" + resource + ".json");

Reply:
you're right, but what is your point? :)